src/test/java/com/mercadopago/BaseClientIT.java

```diff
@@ -0,0 +1,17 @@
+package com.mercadopago;
+
+import org.junit.jupiter.api.BeforeAll;
+
+/** BaseClientIT class. */
+public abstract class BaseClientIT {
+
+  protected static final int DEFAULT_TIMEOUT = 2000;
+
+  protected static final String accessToken = System.getenv("ACCESS_TOKEN");
```

Comment:
```suggestion
  protected static final String accessToken = System.getenv("MP_ACCESS_TOKEN");
```
Pode ser que o integrador precise usar mais de um access token